src/app/pages/profile/profile-sidebar/profile-sidebar.component.scss

```diff
@@ -0,0 +1,112 @@
+.profile {
+  &-sidebar {
+    &-upper {
+      background-color: #f2f3f5;
+    }
+    &-image {
+      width: 70%;
+      height: 90%;
+      overflow: hidden;
+      position: relative;
+      text-align: center;
+    }
+    &-lower {
+      height: 35rem;
+      background-color: #f2f3f5;
+      margin-top: 15%;
+      text-align: center;
+    }
+  }
+  &-image {
+    &-container {
+      // height: 250px;
```

Comment:
remove this commented code here and any other place

Reply:
I fixed this